Source/ViewModel/MLCardFormViewModel.swift

```diff
@@ -215,7 +228,14 @@ final class MLCardFormViewModel {
         let index = unwrappedCardFormFields.firstIndex(where: { $0.property.fieldId() == fieldId }) ?? 0
         let indexWithOffset = min(max(index + offset, 0), unwrappedCardFormFields.count - 1)
         
+        if let currentCardFormField = MLCardFormFields(rawValue: fieldId),
+            currentCardFormField != .identificationTypesPicker {
+            cardFormField.setEnableField(false)
+        }
+        
         if let field = flattenedCardFormFields?[indexWithOffset] {
+            field.setEnableField(true)
```

Comment:
Hi! I believe that this point of the code is ignoring the existence of 2 or more fields in the same screen. That's the case on the "Vencimento, CVV" screen, when the experience has before been both fields enabled at the same time, but now we can edit just one at a time on the same screen, and to navigate/enable the last field we can't just touch on it, but use the "anterior, seguinte" buttons below to enable it. The functionality is preserved, but the user experience has changed.

Maybe a solution here is not to enable just the `indexWithOffset`, but all the indexes that belong to the current screen.

Reply:
@ricardo-dalambert-meli, Can you look at my change?